roles/KubevirtTemplateValidator/defaults/main.yml

```
@@ -2,5 +2,4 @@
 # defaults file for KubevirtTemplateValidator
 wait: true
 validator_image: kubevirt-template-validator
-# this comes from the CR, we must diverge from validator_$SOMETHING standard naming
-version: "{{ validator_tag |default('v0.6.2') }}"
+validator_version: "{{ validator_tag }}"
```

I kind of iterated back to beginning... I think this should be like follows:

```suggestion
validator_version: "{{ validator_tag |default('latest') }}"
```

Or we could name the variable as `version` in case we want to allow CR override. The role should then use just the name like you changed it.

---

works for me.